src_c/polygon.c

```diff
@@ -342,6 +413,8 @@ pg_polygon_copy(pgPolygonObject *self, PyObject *_null)
 }
 
 static struct PyMethodDef pg_polygon_methods[] = {
+    {"normal_polygon", (PyCFunction)pg_polygon_normal_polygon,
```

Comment:
Do we really want this to be part of Polygon? couldn't it be a module method? like `geometry.normal_polygon`? Just a question.

Reply:
it has to do with the object that is why it is a static method